packages/data-store/src/configs.ts

```diff
@@ -1,74 +1,27 @@
-import {DataStoreConfigs, SDKConfigs} from './types/types';
-import {getDataSource} from './typeorm';
-import {SDKConfigsEntity} from './typeorm/entities/sdk-configs.entity';
-
-export async function getSDKConfigs(
-  options: DataStoreConfigs,
-): Promise<SDKConfigs> {
-  let result = await getDataSource().getRepository(SDKConfigsEntity).find();
-
-  if (result.length === 1) {
-    return result[0];
-  }
-
-  if (result.length > 1) {
-    throw new Error('More than one SDKConfigsEntity found');
-  }
-
-  const sdkConfigs = {
-    id: 'sdk-configs',
-    activeWallet: options.sdkConfigs.activeWallet,
-    version: null,
-  };
-
-  await getDataSource().getRepository(SDKConfigsEntity).save(sdkConfigs);
-
-  return sdkConfigs;
-}
-
-export const defaultDataStoreConfigs: DataStoreConfigs = {
-  sdkConfigs: {
-    activeWallet: 'mainnet',
-  },
+import {DataStoreConfigs} from './types';
+import {genericDocumentNetworkResolver} from './document-network-resolver';
+
+export const DEFAULT_CONFIGS: DataStoreConfigs = {
+  defaultNetwork: 'mainnet',
```

Comment:
if we had to support a completely different set of networks (e..g in a white label build for a private version of Dock) how would we override the defaults if they're hard-coded here?

Reply:
No, I've created an example of how to do that on this commit: https://github.com/docknetwork/react-native-sdk/pull/115/commits/3e99e343b144e0eb17d7e9db1dd43fba748d184b

Added a new integration test to cover custom networks and JWT-VC